link-crypto/Cargo.toml

```diff
@@ -33,6 +32,11 @@ features = ["std", "derive"]
 path = "../git-ext"
 features = ["serde", "minicbor"]
 
+[dependencies.radicle-keystore]
+git = "https://github.com/radicle-dev/radicle-keystore"
+rev = "2a28dd785f7cd360cdc962e4d343fa9f7ac39814"
```

Comment:
Although we should probably wait for merging until a minor release is made[0]

[0]: https://github.com/ZcashFoundation/ed25519-zebra/issues/45#issuecomment-820181394

Reply:
Would be preferable, I think, for the upstream release to be made. They said they'd make a decision tomorrow :)